src/views/pages/dashboard/index.js

```diff
@@ -5,11 +5,11 @@ import { Grid } from "@mui/material";
 
 // project imports
 import EarningCard from "./components/EarningCard";
-import PopularCard from "./components/PopularCard";
-import TotalOrderLineChartCard from "./components/TotalOrderLineChartCard";
-import TotalIncomeDarkCard from "./components/TotalIncomeDarkCard";
-import TotalIncomeLightCard from "./components/TotalIncomeLightCard";
-import TotalGrowthBarChart from "./components/TotalGrowthBarChart";
```

Comment:
Changes to this file seem unrelated to FE-12, please move to correct Issue Branch if you would like to keep these changes, otherwise remove the changes to this file from this PR